File: Beecrowd/2221 Batalha de Pomekons/main.c
#include <stdio.h>

int main(){
    int t, b, atq1, def1, lvl1, res1, atq2, def2, lvl2, res2;

    scanf("%d", &t);

    while(t--){
        scanf("%d", &b);
        scanf("%d %d %d", &atq1, &def1, &lvl1);
        scanf("%d %d %d", &atq2, &def2, &lvl2);

        if(lvl1 % 2 == 0)
            res1 = ((atq1 + def1) / 2) + b;
        else
            res1 = (atq1 + def1) / 2;

        if(lvl2 % 2 == 0)
            res2 = ((atq2 + def2) / 2) + b;
        else
            res2 = (atq2 + def2) / 2;

        if(res1 == res2)
            printf("Empate\n");
        else if(res1 > res2)
            printf("Dabriel\n");
        else
            printf("Guarte\n");
    }

    return 0;
}

File: Beecrowd/2221 Batalha de Pomekons/main.py
t = int(input())

for _ in range(t):
    b = int(input())
    jog1 = [int(_) for _ in input().split()]
    jog2 = [int(_) for _ in input().split()]

    if jog1[2] % 2 == 0:
        res1 = ((jog1[0] + jog1[1]) / 2) + b
    else:
        res1 = (jog1[0] + jog1[1]) / 2

    if jog2[2] % 2 == 0:
        res2 = ((jog2[0] + jog2[1]) / 2) + b
    else:
        res2 = (jog2[0] + jog2[1]) / 2

    if res1 == res2:
        print("Empate")
    elif res1 > res2:
        print("Dabriel")
    else:
        print("Guarte")